tests/python/test_grammar_parser.py

```diff
@@ -432,6 +432,7 @@ def test_space():
     expected = """root ::= (("abcde") | ("f") | ("g"))
 """
     grammar = xgr.Grammar.from_ebnf(before)
+    grammar = GrammarFunctor.byte_string_fuser(grammar)
```

Comment:
Ditto, we can update the expected grammar with the result without byte_string_fuser

And add another test_grammar_functor.py to check the functors, such as byte_string_fuser and test_repetition_normalizer